compiler/x/codegen/OMRLinkage.hpp

```diff
@@ -45,31 +45,7 @@ namespace OMR { typedef OMR::X86::Linkage LinkageConnector; }
 #include "infra/Annotations.hpp"
 #include "infra/Assert.hpp"
 #include "x/codegen/X86Ops.hpp"
-
-// linkage properties
-#define CallerCleanup                              0x0001
-#define RightToLeft                                0x0002
-#define IntegersInRegisters                        0x0004
-#define LongsInRegisters                           0x0008
-#define FloatsInRegisters                          0x0010
-#define EightBytePointers                          0x0020
-#define EightByteParmSlots                         0x0040
-#define LinkageRegistersAssignedByCardinalPosition 0x0080
-#define CallerFrameAllocatesSpaceForLinkageRegs    0x0100
-#define AlwaysDedicateFramePointerRegister         0x0200
-#define NeedsThunksForIndirectCalls                0x0400
-#define UsesPushesForPreservedRegs                 0x0800
-#define ReservesOutgoingArgsInPrologue             0x1000
-#define UsesRegsForHelperArgs                      0x2000
-
-// register flags
-#define Preserved                   0x01
-#define IntegerReturn               0x02
-#define IntegerArgument             0x04
-#define FloatReturn                 0x08
-#define FloatArgument               0x10
-#define CallerAllocatesBackingStore 0x20
-#define CalleeVolatile              0x40
+#include "x/codegen/OMRX86LinkageProperties.hpp"
```

Comment:
If the linkage properties header isn't going to be made "extensible" then you don't need the `OMR` prefix.  Perhaps just call it `X86LinkageProperties.hpp`.

Reply:
The `x/` shouldn't be used unless necessary to disambiguate filenames in different directories.

Reply:
Point noted!